rtfs.go

```diff
@@ -137,3 +157,8 @@ func (im *IpfsManager) CustomRequest(ctx context.Context, url, commad string,
 
 	return resp, nil
 }
+
+// SwarmConnect is use to open a connection a one or more ipfs nodes
+func (im *IpfsManager) SwarmConnect(addrs ...string) error {
+	return im.shell.SwarmConnect(context.Background(), addrs...)
```

Comment:
we should allow `context` to be passed in - this allows cancels (`context.WithCancel`) or timeouts, or even additional context headers

Reply:
ah yes good catch.